rasa/shared/utils/schemas/domain.yml

```diff
@@ -51,7 +51,9 @@ mapping:
         type: "map"
         mapping:
           required_slots:
-            type: "map"
+            type: "seq"
```

Comment:
Do we also need to add a section for the new global mappings?

Reply:
Ah yes, definitely! I'll add a section here right away.